### PR TITLE
Unicycler/STAR support + v0.7 features

### PR DESCRIPTION
**Pull request by bcai2**
* Adds support for `unicycler` and `STAR` in the R client.
* Adds the ability to select specific input files interactively (e.g., for long reads), as well as selecting multiple input files interactively for the same job.
* Adds handling for non-essential arguments like `read_one`. If left as NULL, they are ignored so that defaults in the python client are not clobbered.
* Adds support for `kraken2` DB choices.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: toolchest
 Title: R Client for Toolchest
-Version: 0.5.0
+Version: 0.7.0
```
Comment (lebovic) [inline DESCRIPTION:3]: Are you thinking we match minor but not patch numbers Python <-> R?

File: R/tools.R [hunk 2]
```diff
@@ -35,34 +42,89 @@ bowtie2 <- function(tool_args = "", inputs = NULL, output_path = NULL, database_
...
 #'
 #' @export
 cutadapt <- function(tool_args, inputs = NULL, output_path = NULL) {
   inputs <- .validate.inpath(inputs)
   output_path <- .validate.outpath(output_path)
-  toolchest_client$cutadapt(inputs, output_path, tool_args)
+  toolchest_args <- list(
+    inputs = inputs,
+    output_path = output_path,
+    tool_args = tool_args
```
Comment (lebovic) [inline R/tools.R:55]: 👌

File: R/tools.R [hunk 3]
```diff
@@ -35,34 +42,89 @@ bowtie2 <- function(tool_args = "", inputs = NULL, output_path = NULL, database_
...
+  )
+  .do.toolchest.call(toolchest_client$cutadapt, toolchest_args)
 }
 
 #' Kraken 2 Client
 #'
 #' Starts a query for Kraken 2 using Toolchest.
 #'
-#' (Currently, only single .fastq inputs are supported.)
-#'
```
Comment (lebovic) [inline R/tools.R:53]: 🎉

File: R/tools.R [hunk 4]
```diff
@@ -35,34 +42,89 @@ bowtie2 <- function(tool_args = "", inputs = NULL, output_path = NULL, database_
...
+#' @note If the database is unspecified, Kraken 2 will use the standard
+#' database, using index files generated by Ben Langmead's lab at
+#' \url{https://benlangmead.github.io/aws-indexes/k2}.
 #'
 #' @export
-kraken2 <- function(tool_args = "", inputs = NULL, output_path = NULL) {
-  inputs <- .validate.inpath(inputs)
+kraken2 <- function(tool_args = "", inputs = NULL, output_path = NULL,
+                    database_name = NULL, database_version = NULL) {
+  inputs <- .validate.inpath(inputs, choose_multiple = TRUE)
```
Comment (lebovic) [inline R/tools.R:80]: Did you test Kraken with multiple inputs?

File: README.md [hunk 5]
```diff
@@ -47,6 +47,7 @@ Toolchest currently supports the following tools:
 -   Bowtie 2 (`bowtie2`)
 -   Cutadapt (`cutadapt`)
 -   Kraken 2 (`kraken2`)
+-   Unicycler (`unicycler`)
```
Comment (lebovic) [inline README.md:50]: Missing STAR?

**Review by lebovic — APPROVED** [review]
👍 

Can't test on my current environment (bad local connection), but will test prior to merge to main

File: DESCRIPTION (hunk 1 above)
Comment (bcai2) [reply DESCRIPTION:3]: That's what I was thinking; if we need to update something on one side but not the other, things could get out of hand.

File: R/tools.R (hunk 4 above)
Comment (bcai2) [reply R/tools.R:80]: Kraken with multiple inputs is currently disabled on the Python side. We'll eventually want to support this, so I enabled this option, but we can remove it for now and re-add it once it's good to go on Python.

File: README.md (hunk 5 above)
Comment (bcai2) [reply README.md:50]: Good point, will edit

File: R/tools.R (hunk 4 above)
Comment (lebovic) [reply R/tools.R:80]: Sounds good

Comment (lebovic) [reply R/tools.R:80]: (meaning we should probably disable until enabled on the Python side)

**Pull request merged**